packages/core/build/config.js

```diff
@@ -114,6 +114,10 @@ const copyConfig = base => {
             from: path.resolve(__dirname, '../src/public/images/common/static_images/'),
             to: 'public/images/common',
         },
+        {
+            from: path.resolve(__dirname, '../src/public/video/'),
```

Comment:
Removed video from images/common/static_images to a separate folder

Reply:
`'../src/public/videos/'`

Reply:
Will do, thank you. But PR is not finished yet, I'm waiting for the rest videos)